data_steward/retraction/retract_utils.py

```diff
@@ -79,6 +84,65 @@ def get_src_id(mapping_type):
     return src_id
 
 
+def get_datasets_list(project_id, dataset_ids):
+    # initialize list of all datasets in project
```

Comment:
Please add a doc string detailing what this function is for.  As is, this function will raise an `Attribute Error` if an object without the `.split()` attribute is passed into `dataset_ids`.  You can either check for this in the code and do something with it or (I might do this) add the raises AttributeError information in the function doc string not make any code changes.